java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/dao/JobId.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 4Paradigm
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com._4paradigm.openmldb.taskmanager.dao;
+
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.LongAdder;
+
+public class JobId {
+    private static final AtomicLong jobId = new AtomicLong();
```

Comment:
Need to recovery from zookeeper so that it will not return same value after process restarts

Reply:
Resolved